src/index.js

```diff
@@ -11,10 +11,11 @@ function defaultAssignEntity(normalized, key, entity) {
 function visitObject(obj, schema, bag, options) {
   const { assignEntity = defaultAssignEntity } = options;
 
-  let normalized = {};
+  const defaults = schema && schema.getDefaults && schema.getDefaults();
+  const schemaAssignEntity = schema && schema.getAssignEntity && schema.getAssignEntity();
+  let normalized = isObject(defaults) ? defaults : {};
```

Comment:
To fix the test I noted, I believe you just need to make a new object that includes the default values:

```js
let normalized = isObject(defaults) ? { ...defaults } : {};
// or
let normalized = isObject(defaults) ? Object.assign({}, defaults) : {};
```